framework/packages/standards/money-market/src/query.rs

```diff
@@ -1,276 +1,156 @@
+pub struct WholeMoneyMarketQuery(
+    pub fn(value: &str) -> Result<Box<dyn MoneyMarketCommand>, MoneyMarketError>,
+    pub MoneyMarketQueryMsg,
+);
 
-mod raw {
-    use cw_asset::AssetInfoBase;
-    /// Possible raw queries to run on the Money Market
-    #[cosmwasm_schema::cw_serde]
-    pub enum MoneyMarketRawQuery {
-        /// Deposited funds for lending
-        UserDeposit {
-            /// User that has deposited some funds
-            user: String,
-            /// Lended asset to query
-            asset: AssetInfoBase<String>,
-            contract_addr: String,
-        },
-        /// Deposited Collateral funds
-        UserCollateral {
-            /// User that has deposited some collateral
-            user: String,
-            /// Collateral asset to query
-            collateral_asset: AssetInfoBase<String>,
-            /// Borrowed asset to query
-            borrowed_asset: AssetInfoBase<String>,
-            contract_addr: String,
-        },
-        /// Borrowed funds
-        UserBorrow {
-            /// User that has borrowed some funds
-            user: String,
-            /// Collateral asset to query
-            collateral_asset: AssetInfoBase<String>,
-            /// Borrowed asset to query
-            borrowed_asset: AssetInfoBase<String>,
-            contract_addr: String,
-        },
-        /// Current Loan-to-Value ratio
-        /// Represents the borrow usage for a specific user
-        /// Allows to know how much asset are currently borrowed
-        CurrentLTV {
-            /// User that has borrowed some funds
-            user: String,
-            /// Collateral asset to query
-            collateral_asset: AssetInfoBase<String>,
-            /// Borrowed asset to query
-            borrowed_asset: AssetInfoBase<String>,
-            contract_addr: String,
-        },
-        /// Maximum Loan to Value ratio for a user
-        /// Allows to know how much assets can to be borrowed
-        MaxLTV {
-            /// User that has borrowed some funds
-            user: String,
-            /// Collateral asset to query
-            collateral_asset: AssetInfoBase<String>,
-            /// Borrowed asset to query
-            borrowed_asset: AssetInfoBase<String>,
-            contract_addr: String,
-        },
-        /// Price of an asset compared to another asset
-        /// The returned decimal corresponds to
-        /// How much quote assets can be bought with 1 base asset
-        Price {
-            quote: AssetInfoBase<String>,
-            base: AssetInfoBase<String>,
-        },
+pub fn err(e: MoneyMarketError) -> AnsHostError {
+    AnsHostError::QueryFailed {
+        method_name: "resolve money market".to_string(),
+        error: cosmwasm_std::StdError::GenericErr { msg: e.to_string() },
     }
 }
 
-mod ans {
-    use crate::MoneyMarketCommand;
-    use abstract_core::objects::AssetEntry;
-    use abstract_sdk::Resolve;
+impl Resolve for WholeMoneyMarketQuery {
```

Comment:
Can we call this struct `MoneyMarketQueryResolveWrapper` or something more explicit on what it's actually used for?

Reply:
Done. Also renamed the execute one to match this name